101-bundle-json.md

```diff
@@ -269,7 +271,8 @@ Fields:
 - `images`: The list of dependent images
   - `description`: The description field provides additional context of the purpose of the image. 
   - `imageType`: The `imageType` field MUST describe the format of the image. The list of formats is open-ended, but any CNAB-compliant system MUST implement `docker` and `oci`. The default is `oci`.
-  - `image`: The REQUIRED `image` field provides a valid reference (REGISTRY/NAME:TAG) for the image. Note that SHOULD be a CAS SHA, not a version tag as in the example above.
+  - `image`: The REQUIRED `image` field provides a valid reference for the image. Note that SHOULD be a CAS SHA, as in the example above, not a version tag.
+  - `originalImage`: The OPTIONAL `originalImage` field provides another valid reference for the image. Note that SHOULD be a CAS SHA, as in the example above, not a version tag. If the `originalImage` field is omitted, its value defaults to that of the `image` field.
```

Comment:
Excellent!

Reply:
Note that the originalImage reference should be grammatically a valid reference, but since it will be replaced at runtime with the "image" value, it can potentially point to a deleted repository.

Reply:
> Note that the originalImage reference should be grammatically a valid reference, but since it will be replaced at runtime with the "image" value, it can potentially point to a deleted repository.

Good point! However, I don't really want to encourage people to delete original images. For example, we'd like our users to be able to see that the digest of the original and new image references are identical so that it's certain that the images consist of the same bits. Deleting the original image would stop them being able to obtain its digest.

Also, we currently say that the run tool MAY replace any original image references, so if we want to allow for original images to be deleted, we'd probably want to strengthen that and say the run tool SHOULD replace those references.